crawler/lookingglass_crawler/spiders/media_spider.py

```
@@ -0,0 +1,107 @@
+from ..items import MediaItem
+
+class MediaSpider(CrawlSpider):
+    name = 'media_spider'
+    
+    def __init__(self, *args, **kwargs):
+        super(MediaSpider, self).__init__(*args, **kwargs)
+        self.start_urls = ['https://www.flickr.com/search/?text=portrait']
+        
+        media_extensions = {
+            'jpg', 'jpeg', 'png', 'gif', 'webp', 'bmp', 'mp4', 
+            'webm', 'avi', 'mov', 'wmv', 'flv', 'mkv'
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)